nemo/collections/asr/metrics/wer.py

```diff
@@ -559,20 +685,32 @@ def _get_word_offsets_subwords_sentencepiece(
         # This is because we always skip the delay the injection of the first sub-word due to the loop
         # condition and check whether built token is ready or not.
         # Therefore without this forced injection, the start_offset appears as off by 1.
-        word_offsets[0]["start_offset"] = offsets[0]["start_offset"]
-
-        # If there are any remaining tokens left, inject them all into the final word offset.
-        # Note: The start offset of this token is the start time of the first token inside build_token.
-        # Note: The end offset of this token is the end time of the last token inside build_token
-        if len(built_token) > 0:
-            word_offsets.append(
-                {
-                    "word": decode_tokens_to_str(built_token),
-                    "start_offset": offsets[-(len(built_token))]["start_offset"],
-                    "end_offset": offsets[-1]["end_offset"],
-                }
-            )
-        built_token.clear()
+        if len(word_offsets) == 0:
```

Comment:
Why was all of this logic modified without my review?? Please remember to add the reviewer of parts of code you see have been modified by them.